### PR TITLE
[CONTRIBUTING.md] Adds instructions for editing the README on GitHub

### LINKED ISSUE #327: Instructions for adding speaker/organizer info using "edit this page" on GitHub directly.
Hi @jlooper ! If you'd like, I'd be happy to contribute to the contributing document with instructions for adding an entry to the README via editing the page on GitHub directly.

I maintain similar kinds of instructions [for the open source project I maintain](https://astro-docs-docs.netlify.app/first-time/4-edit-single-page/#edit-this-page-via-github) and also [in a guide on my personal site](https://rainsberger.ca/blog/contribute-open-source-docs-edit-page-on-github/) for contributors to our documentation site where often people are just making very small content edits. Often, just doing this in the browser is sufficient for those kinds of changes, especially when people aren't necessarily looking to maintain a fork (and, that's in fact how I made my own PR last night).

If you're not interested, no worries; just close this issue!  Just offering!

### PR DESCRIPTION
**Pull request by sarah11918**
This is just a proposal to get the ball rolling and give you something to start from. Please feel free to edit to taste, or leave comments and I will happily revise to your specs!

Thank you again for considering this contribution. I inferred from the care and detail in the existing contributing docs that you want to provide this level of guidance for contributors. I have tried to match this in my instructions, but have no problems with you making whatever changes you like!

Closes #327

### CODE REVIEW

**Review by jlooper — CHANGES_REQUESTED** [review]
hi! this is a much nicer version of a contributing.md file, I like it. The biggest issue I've seen in the past day or so of taking over this repo is people don't add themselves alphabetically! It's kind of a pain. Could you add directions about that along with the request to not dupe names? thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,20 +2,35 @@
...
+7. The maintainers will review your PR and either request changes or approve it.
+
+### Edit the repository locally
+
+1. Fork the repository. Here are the [GitHub instructions for forking a repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+2. In the command line, clone the repository by typing `git clone https://github.com/YOUR-GITHUB-USERNAME/women-tech-speakers-organizers.git`
+3. Then type `cd women-tech-speakers-organizers` to go into the project's directory.
+4. Create a branch and switch to that new branch by using the following command: `git checkout -b new-branch-name`
+5. Update the `README.md` with your newsletter suggestion(s).
    - Append the title and link to the newsletter, to the list in the appropriate category
```
Comment (sarah11918) [inline CONTRIBUTING.md:28]:
```suggestion
5. Update the `README.md` with your new suggestion.
   - Scroll down to the appropriate location in the file to add your suggestion. Suggestions are organized first by country, and then **by last name alphabetically**.
   - Use the [format](#formatting) provided as a template for adding your suggestion to the document.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,20 +2,35 @@
...
-3. In the command line, clone the repository by typing `git clone https://github.com/YOUR-GITHUB-USERNAME/women-tech-speakers-organizers.git`
-4. Then type `cd women-tech-speakers-organizers` to go into the project's directory.
-5. Create a branch and switch to that new branch by using the following command: `git checkout -b new-branch-name`
-6. Update the `README.md` with your newsletter suggestion(s).
+You can edit this file [using GitHub's web interface](#edit-the-readme-on-github), or fork the repository and [work locally in your code editor](#edit-the-repository-locally).
+
+### Edit the README on GitHub
+
+1. Navigate to the README file in the repository and click the pencil icon to "edit file", or navigate to [the URL to edit this file directly](https://github.com/fempire/women-tech-speakers-organizers/edit/main/README.md).
+2. Scroll down to the appropriate location in the file to add your suggestion. Suggestions are organized first by country, and then by last name alphabetically.
```
Comment (sarah11918) [inline CONTRIBUTING.md:14]:
```suggestion
2. Scroll down to the appropriate location in the file to add your suggestion. Suggestions are organized first by country, and then **by last name alphabetically**.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,20 +2,35 @@
...
 ## How to contribute
 
-1. Check the `README` to see if your suggestion exists or not.
-2. Fork the repository. Here are the [GitHub instructions for forking a repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
-3. In the command line, clone the repository by typing `git clone https://github.com/YOUR-GITHUB-USERNAME/women-tech-speakers-organizers.git`
-4. Then type `cd women-tech-speakers-organizers` to go into the project's directory.
-5. Create a branch and switch to that new branch by using the following command: `git checkout -b new-branch-name`
-6. Update the `README.md` with your newsletter suggestion(s).
+You can edit this file [using GitHub's web interface](#edit-the-readme-on-github), or fork the repository and [work locally in your code editor](#edit-the-repository-locally).
+
```
Comment (sarah11918) [inline CONTRIBUTING.md:10]:
```suggestion

**Please add new names in alphabetical order by last name, under the appropriate category and country.**

```

**Review by sarah11918 — COMMENTED** [review]
Since it's a pain point, I added a note and then also bolded in each of the two sets of instructions.

I figured rather than two bolded notes right next to each other, I spread them out so they'd stand out from each other a little better. Maybe not perfect, but I think they'll be forced to get the message.

**Pull request merged**